Trustroots/trustroots
# Remove logged-out search page

#### Proposed Changes

Removes logged-out search page. I started looking at turning this into React but realized it isn't really much in use:

**Before**

<img width="1272" alt="Screenshot 2020-12-20 at 19 46 45" src="https://user-images.githubusercontent.com/87168/102720474-4400e300-42fd-11eb-9546-093017afdee3.png">

**After**

![image](https://user-images.githubusercontent.com/87168/102720653-516a9d00-42fe-11eb-880b-67992aaadad1.png)


It was created because we linked into search directly from Hitchwiki, and since Trustroots was new to most folks in Hitchwiki it was appropriate to show some explanation of what TR is. By now most Hitchwiki folks should know it pretty well. :-) 

It still is kinda useful for that, but at the same time, I like to simplify and have less code around. :-)

<img width="342" alt="Screenshot 2020-12-20 at 19 50 54" src="https://user-images.githubusercontent.com/87168/102720472-3ba8a800-42fd-11eb-9383-19d3d53609d4.png">


#### Testing Instructions

* As logged out, open `/search`
* You should see a regular login form instead of special invite to login
